xml/System/String.xml

```diff
@@ -4129,13 +4134,15 @@ You can download the [Sorting Weight Tables](https://www.microsoft.com/download/
  This method performs an ordinal (case-sensitive and culture-insensitive) comparison. The search begins at the first character position of this string and continues through the last character position.  
   
  To determine whether a string contains a specified substring by using something other than ordinal comparison (such as culture-sensitive comparison, or ordinal case-insensitive comparison), you can create a custom method. The following example illustrates one such approach. It defines a <xref:System.String> extension method  that includes a <xref:System.StringComparison> parameter and indicates whether a string contains a substring when using the specified form of string comparison.  
-  
+ 
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```

Comment:
Move this to the examples section since it comes first in the page

Reply:
Done